tasks/databases.rake

```diff
@@ -311,6 +417,22 @@ namespace :data do
   end
 end
 
+def print_memory_usage
```

Comment:
also, maybe it would be good to move this method into a separate class (and file) - database.rake is already pretty big